app/email_test.go

```diff
@@ -133,3 +135,49 @@ func TestSendAdminUpgradeRequestEmailOnJoin(t *testing.T) {
 	require.NotNil(t, err)
 	assert.Equal(t, err.Id, "app.email.rate_limit_exceeded.app_error")
 }
+
+func TestSendInviteEmails(t *testing.T) {
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+
+	inbucket_host := os.Getenv("CI_INBUCKET_HOST")
+	if inbucket_host == "" {
+		inbucket_host = "localhost"
+	}
+	inbucket_port := os.Getenv("CI_INBUCKET_SMTP_PORT")
+	if inbucket_port == "" {
+		inbucket_port = "10025"
+	}
```

Comment:
I can see this logic being used again in the future. Wondering if it makes sense to extract it in an internal test util function.